src/reporters/buffer-reporter.js

```diff
@@ -27,4 +27,12 @@ export default class BufferReporter extends JSONReporter {
   getBuffer(): Buffer {
     return this._buffer;
   }
+
+  getBufferText(): String {
+    return this._buffer.map(({data}) => String(data)).join(``);
+  }
+
+  getBufferJson(): any {
```

Comment:
This name is misleading since the returned thing is not JSON. Also, I don't understand the need to go through casting things to a JSON string and back into an object. Why not use the `JSONReporter` at this point and just parse the whole thing?

Reply:
We already have `readJson` in a few places, which does not return json either ;) What would you suggest?

> Why not use the JSONReporter at this point and just parse the whole thing?

- The JSON reporter wraps what is written in irrelevant fields (`type` and `data`)
- The `data` field is a string, so it would have to be parsed anyway.

Reply:
I'm sad because we don't have a good way of ingesting JSON output easily :(